src/tests/test_calculator.cc

```diff
@@ -46,8 +46,9 @@ BOOST_AUTO_TEST_CASE(load_defaults_test) {
       std::ofstream defaults("calculators/xml/testcalc.xml");
       defaults << "<options>\n"
                << "<testcalc>\n"
-               << "<option1>0</option1>\n"
-               << "<option2>3.141592</option2>\n"
+               << "<option0 choices=\"foo,bar\">foo</option0>\n"
```

Comment:
we need also test cases for the error cases.

Reply:
See next commit